README.md

```diff
@@ -857,17 +841,9 @@ you can do so.
       but this list is expected to grow over time as vLLM updates.
       ```yaml
       ecsModels:
-        - modelName: mistralai/Mistral-7B-Instruct-v0.2
-          modelId: mistral7b-vllm
-          deploy: false
-          modelType: textgen # can also be 'embedding'
-          streaming: true # remove option if modelType is 'embedding'
-          instanceType: g5.xlarge
-          inferenceContainer: vllm # vLLM-specific config
-          containerConfig:
-            image:
-              baseImage: vllm/vllm-openai:v0.5.0 # vLLM-specific config
-              path: lib/serve/ecs-model/vllm # vLLM-specific config
+        - modelName: your-model-name
+          inferenceContainer: tgi # vLLM-specific config
+          baseImage: ghcr.io/huggingface/text-generation-inference:1.0.2 # vLLM-specific config
```

Comment:
vLLM references on tgi options. also that container tag is way too old. just for examples, everything for TGI should be >=1.4.0